### PR TITLE
Chore(New_Exp): Add docs for azure instance terminate chaos

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <udit@chaosnative.com>

**What this PR does / why we need it**:

This PR is for the addition of docs for the new azure experiment that is azure instance terminate.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [x] Fixes https://github.com/litmuschaos/litmus/issues/2418
-   [x] Signed the commit for [DCO](https://github.com/litmuschaos/litmus-docs/blob/master/CONTRIBUTING.md#sign-your-work) check to be passed
-   [x] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/azure-instance-terminate.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  name: azure-instance-terminate-sa
+subjects:
+- kind: ServiceAccount
+  name: azure-instance-terminate-sa
+  namespace: default
+```
+
+### Prepare ChaosEngine
+
+- Provide the application info in `spec.appinfo`
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]: Can we add to note to mention this step as optional?

File: docs/azure-instance-terminate.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+<table>
+  <tr>
+    <th> Variables </th>
+    <th> Description </th>
+    <th> Specify In ChaosEngine </th>
+    <th> Notes </th>
+  </tr>
+  <tr> 
+    <td> AZURE_INSTANCE_NAME </td>
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]: Can we add details for Region env as well?

File: docs/azure-instance-terminate.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
+  namespace: default
+spec:
+  annotationCheck: 'false'
+  engineState: 'active'
+  chaosServiceAccount: azure-instance-terminate-sa
+  monitoring: false
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]: We need to remove this field

File: docs/azure-instance-terminate.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+            # set chaos duration (in sec) as desired
+            - name: TOTAL_CHAOS_DURATION
+              value: '60'
+
+             # Instance ID of the target ec2 instance
+             # If not provided it will select randomly from the region
+            - name: EC2_INSTANCE_ID
+              value: ''
+              
+            # provide the region name of the instace
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]:
```suggestion
            # provide the region name of the instance
```

File: docs/azure-instance-terminate.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+  `az vm list -d -o table --query "[?name=='<vm-name>']"` (vm name without <>)
+
+-  You can also use Azure console to keep a watch over the instance state.   
+
+### Check Chaos Experiment Result
+
+- Check whether the application is resilient to the azure-instance-terminate, once the experiment (job) is completed. The ChaosResult resource name is derived like this: `<ChaosEngine-Name>-<ChaosExperiment-Name>`.
+
+  `kubectl describe chaosresult nginx-chaos-azure-instance-terminate -n <application-namespace>`
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]:
```suggestion
  `kubectl describe chaosresult nginx-chaos-azure-instance-terminate -n <chaos-namespace>`
```

File: docs/azure-instance-terminate.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+  <tr> 
+    <td> TOTAL_CHAOS_DURATION </td>
+    <td> The time duration for chaos insertion (sec) </td>
+    <td> Optional </td>
+    <td> Defaults to 30s </td>
+  </tr>
+  <tr>
+    <td> RESOURCE_GROUP </td>
+    <td> The resource group of the target instance</td>
+    <td> Mandatory </td>
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]: Can we move Mandatoy fields upward? order should be Mandatory fields followed by Optional field

File: docs/azure-instance-terminate.md (hunk 2 above)
Comment (uditgaurav) [reply docs/azure-instance-terminate.md]: we are not using region in Azure! We are targeting instance on basis on `RESOURCE_GROUP`

File: docs/azure-instance-terminate.md (hunk 3 above)
Comment (uditgaurav) [reply docs/azure-instance-terminate.md]: Needs the change in chart.

File: docs/azure-instance-terminate.md (hunk 4 above)
Comment (uditgaurav) [reply docs/azure-instance-terminate.md]: It is also coming from charts

File: docs/azure-instance-terminate.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+            - name: TOTAL_CHAOS_DURATION
+              value: '60'
+
+             # Instance ID of the target ec2 instance
+             # If not provided it will select randomly from the region
+            - name: EC2_INSTANCE_ID
+              value: ''
+              
+            # provide the region name of the instace
+            - name: REGION
```
Comment (ispeakc0de) [inline docs/azure-instance-terminate.md]: We can remove this

File: docs/azure-instance-stop.md [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+    <td> Termination of an azure instance for a certain chaos duration</td>
+    <td> EKS </td>
+  </tr>
+</table>
+
+## Prerequisites
+
+- Ensure that Kubernetes Version > 1.13
+- Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
+- Ensure that the `azure-instance-stop` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace If not, install from [here](https://hub.litmuschaos.io/api/chaos/master?file=charts/kube-aws/azure-instance-stop/experiment.yaml)
```
Comment (ispeakc0de) [inline docs/azure-instance-stop.md]:
```suggestion
- Ensure that the `azure-instance-stop` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace If not, install from [here](https://hub.litmuschaos.io/api/chaos/master?file=charts/azure/azure-instance-stop/experiment.yaml)
```

File: docs/azure-instance-stop.md [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+
+  `kubectl patch chaosengine <chaosengine-name> -n <namespace> --type merge --patch '{"spec":{"engineState":"active"}}'`
+
+### Check Chaos Experiment Result
+
+- Check whether the application is resilient to the azure-instance-stop, once the experiment (job) is completed. The ChaosResult resource name is derived like this: `<ChaosEngine-Name>-<ChaosExperiment-Name>`.
+
+  `kubectl describe chaosresult nginx-chaos-azure-instance-stop -n <chaos-namespace>`
+
+### EC2 Stop Experiment Demo
```
Comment (avaakash) [inline docs/azure-instance-stop.md]:
```suggestion
### Azure Instance Stop Experiment Demo
```

File: docs/azure-instance-stop.md [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+    <td>  </td>
+  </tr>
+  <tr>
+    <td> RESOURCE_GROUP </td>
+    <td> The resource group of the target instance</td>
+    <td> Mandatory </td>
+    <td> </td>
+  </tr> 
+  <tr> 
+    <td> TOTAL_CHAOS_DURATION </td>
```
Comment (ispeakc0de) [inline docs/azure-instance-stop.md:167]: we can add chaos interval details as well

File: docs/azure-instance-stop.md [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+    <td> The time duration for chaos insertion (sec) </td>
+    <td> Optional </td>
+    <td> Defaults to 30s </td>
+  </tr>
+  <tr>
+    <td> RAMP_TIME </td>
+    <td> Period to wait before injection of chaos in sec </td>
```
Comment (ispeakc0de) [inline docs/azure-instance-stop.md:180]:
```suggestion
    <td> Period to wait before/after injection of chaos in sec </td>
```

File: docs/azure-instance-stop.md [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+  experiments:
+    - name: azure-instance-stop
+      spec:
+        components:
+          env: 
+            # set chaos duration (in sec) as desired
+            - name: TOTAL_CHAOS_DURATION
+              value: '30'
+
+            # provide the target instance name
```
Comment (avaakash) [inline docs/azure-instance-stop.md:224]: Can we add CHAOS_INTERVAL here

**Review by avaakash — DISMISSED** [review]

**Review by ispeakc0de — DISMISSED** [review]

**Review by ispeakc0de — DISMISSED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**